App.js

```diff
@@ -31,10 +31,7 @@ export default class App extends Component<{}> {
   }
 
   launchSDK() {
-    NativeModules.OnfidoSDK.startSDK(
-      (applicantId) => { this.setTextContent("Verification complete", "To perform another verification, press \"Launch\"") },
-      (errorCause) => { this.setTextContent("Flow not finished", "To try again, press \"Launch\"") }
-    );
+    NativeModules.OnfidoSDK.startSDK();
```

Comment:
can we maybe have 2 different buttons to launch both SDKs? these callbacks are needed to update the UI after the flow completion

Reply:
@sergiomarquesmoura this PR is not to merge yet it is incomplete (on purpose) but here for visibility.

Reply:
added `[DO NOT MERGE - INCOMPLETE]` for calrity